### ISSUE TITLE
Popup injectors error on v0.21

### ISSUE THREAD

**Issue by aarepuu** [soal/vue-mapbox#91]
Hi,

Just updated to vue-mapbox v0.21 and getting an error on the popup component. I'm using one popup for the map and moving it around based on the coordinates provided from an event and then setting the popup on the go. Example code I'm using is available under issue #66. It's not breaking anything, but just wanted to flag it up.

The error I'm getting now in the console is as follows:
```
vue.esm.js?efeb:628 [Vue warn]: Injection "marker" not found

found in

---> <Popup> at Popup.vue
       <GeoPopup> at src/components/geo/GeoPopup.vue
         <GlMap> at GlMap.vue
           <GeoMap> at src/components/geo/GeoMap.vue
             <SimpleLayout> at src/layouts/SimpleLayout.vue
               <Main> at src/views/base/Main.vue
                 <App> at src/App.vue
                   <Root>
```
Cheers

**Comment by soal** [comment]
-------- Исходное сообщение --------
Вкл 21 февр. 2019 г., 13:10, Aare Puussaar пишет:

**Comment by soal** [comment]
Fixed in [0.2.2](https://github.com/soal/vue-mapbox/releases/tag/0.2.2)
